lib/src/map/map.dart

```diff
@@ -20,8 +20,8 @@ class MapControllerImpl implements MapController {
     }
   }
 
-  void move(LatLng center, double zoom) {
-    _state.move(center, zoom);
+  void move(LatLng center, double zoom, [bool hasGesture = false]) {
```

Comment:
consider making this a named optional rather than an unnamed optional param

Reply:
why?

Reply:
just something to consider for readability